jiant/scripts/postproc/xtreme/xtreme_runconfig_writer.py

```diff
@@ -0,0 +1,114 @@
+import jiant.proj.main.scripts.configurator as configurator
+import jiant.utils.zconf as zconf
+import jiant.utils.python.io as py_io
+from jiant.utils.python.datastructures import get_unique_list_in_order
+
+
+LANGS_DICT = {
+    "xnli": "ar bg de el en es fr hi ru sw th tr ur vi zh".split(),
+    "pawsx": "de en es fr ja ko zh".split(),
+    "udpos": "af ar bg de el en es et eu fa fi fr he hi hu id it ja ko mr"
+    " nl pt ru ta te tr ur vi zh".split(),
+    "panx": "af ar bg bn de el en es et eu fa fi fr he hi hu id it ja jv ka kk ko ml mr ms my"
+    " nl pt ru sw ta te th tl tr ur vi yo zh".split(),
+    "xquad": "ar de el en es hi ru th tr vi zh".split(),
+    "mlqa": "ar de en es hi vi zh".split(),
+    "tydiqa": "ar bn en fi id ko ru sw te".split(),
+    "bucc2018": "de fr ru zh".split(),
+    "tatoeba": "af ar bg bn de el es et eu fa fi fr he hi hu id it ja jv ka kk ko ml mr"
+    " nl pt ru sw ta te th tl tr ur vi zh".split(),
+}
+TRAIN_TASK_DICT = {
+    "xnli": "mnli",
+    "pawsx": "pawsx_en",
+    "udpos": "udpos_en",
+    "panx": "panx_en",
+    "xquad": "squad_v1",
+    "mlqa": "squad_v1",
+    "tydiqa": "tydiqa_en",
+}
+TRAINED_TASKS = ["xnli", "pawsx", "udpos", "panx", "xquad", "mlqa", "tydiqa"]
+UNTRAINED_TASKS = ["bucc2018", "tatoeba"]
```

Comment:
Background info comment would be helpful here.

Reply:
See above: users of XTREME will be familiar with it.